features/step_definitions/page_section_steps.rb

```diff
@@ -44,52 +45,44 @@
 
 Then(/^that section is there too$/) do
```

Comment:
This one is vague / strangely named if you want to add that to your refactor.  It's used in:
```gherkin

  Scenario: Designate a section of a page
    When I navigate to the home page
    Then I can see elements in the section
    When I navigate to another page
    Then that section is there too

```